ci/gitlab/bin/quicc_yml_gen_pipelines.py

```diff
@@ -25,13 +25,14 @@
         pipe = libtest_pipeline(c)
         pipe.write()
 
-    # Model stability without timing pipelines
-    model_stability_confs = [
-        config('mpi-petsc', 'alps-zen2', 'baseimage_petsc')
-        ]
-    for c in model_stability_confs:
-        pipe = model_stability_pipeline(c)
-        pipe.write()
+    #/// \todo this is broken
```

Comment:
What's broken?

Reply:
the logic for the base image changed, it should be fixed and the stability pipeline restore but not in this PR.
I created and issue #14 .